src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -922,8 +922,8 @@ def list(self, fs: str, path: str = '/', recursive: bool = True):
         for snap in snapshot_schedule_list:
             current_path = snap.strip().split(' ')[0]
             error_code, status_out, err = mgr.remote('snap_schedule', 'snap_schedule_get',
-                                                     current_path, fs, 'plain')
-            output.append(json.loads(status_out))
+                                                     current_path, fs, None, None, 'json')
+            output = output + json.loads(status_out)
```

Comment:
it could be because you intialized output as a list and then here you are appending the json string to it so essentially the type of the variable is being changed here.

Reply:
I had to make this change in order for it to work with snap schedule of the same path. The output from the module is very strange and it doesn't match  the output of the CLI command. Loading the data from the module as 'plan' caused me loads of problems.